examples/tv-app/android/App/app/src/main/java/com/tcl/chip/chiptvserver/service/MatterServant.java

```diff
@@ -92,6 +92,8 @@ public void init(@NonNull Context context) {
             new ChipMdnsCallbackImpl(),
             new DiagnosticDataProviderImpl(applicationContext));
 
+    chipPlatform.updateCommissionableDataProviderData(null, null, 0, 20202021, 0xF00);
```

Comment:
please add comments on what arguments mean. During code review it is hard for me to figure out what null,null,0 are.

Should we have constants for the 20202021 and 0xF00 constants for readability?

Reply:
fixed, added comment on API and constants for 20202021 and 0xF00